### PR TITLE
League submission usability improvements

### PR DESCRIPTION
**Pull request by zachstence**
https://app.clickup.com/t/2z0jnd9

- League play schedule in accordions
  - Current week shown at the top and expanded by default
  - All other weeks shown below and collapsed by default
- Clean up fixture submission/ratification page
  - Use resolver/endpoint for `canSubmit`
  - Use resolver/endpoint for `canRatify`
  - Show submit/ratify/completed button
- Create Match player guard
- Fix store types
- Remove debug logs
- Add simple accordion atom
- Remove dead scrim code for `endScrim` and `ScrimStatus.RATIFYING`

---

screencapture-localhost-3000-league-2022-09-29-20_20_08

submit and completed

ratify and completed

### CODE REVIEW

**Comment by 0c370t** [comment]
Task linked: [CU-2z0jnd9 League Play Usability Improvements](https://app.clickup.com/t/2z0jnd9)

File: microservices/submission-service/src/replay-submission/replay-submission-util.service.ts [hunk 1]
```diff
@@ -105,4 +105,88 @@ export class ReplaySubmissionUtilService {
...
+        }
+
+        if (!submission.validated) {
+            return {
+                canRatify: false,
+                reason: "The submission has not been validated",
+            };
+        }
+
+        if (submissionIsScrim(submissionId)) {
```
Comment (bwpikaard) [inline microservices/submission-service/src/replay-submission/replay-submission-util.service.ts:126]: submission.type?

**Review by bwpikaard — DISMISSED** [review]

File: microservices/submission-service/src/replay-submission/replay-submission-util.service.ts (hunk 1 above)
Comment (zachstence) [reply microservices/submission-service/src/replay-submission/replay-submission-util.service.ts:126]: Meh, we could. This structure matches the `canSubmitReplays` function (these are very similar), so I'm gonna leave it

File: clients/web/src/lib/api/queries/league/LeagueFixture.store.ts [hunk 2]
```diff
@@ -11,19 +11,32 @@ export interface FixtureFranchise {
...
+    id: number;
     homeFranchise: FixtureFranchise;
     awayFranchise: FixtureFranchise;
 
     scheduleGroup: {description: string;};
     matches: Array<{
         id: number;
-        skillGroup: {description: string; ordinal: number;};
+        skillGroup: {
+            ordinal: number;
```
Comment (bwpikaard) [inline clients/web/src/lib/api/queries/league/LeagueFixture.store.ts:23]: Two ordinals?

**Review by bwpikaard — CHANGES_REQUESTED** [review]

File: clients/web/src/lib/api/queries/league/LeagueFixture.store.ts (hunk 2 above)
Comment (bwpikaard) [reply clients/web/src/lib/api/queries/league/LeagueFixture.store.ts:23]: (check below, in profile)

Comment (zachstence) [reply clients/web/src/lib/api/queries/league/LeagueFixture.store.ts:23]: [a53d160](https://github.com/SprocketBot/sprocket/pull/309/commits/a53d16023a3fc9977307129a2065d693fbcb3ba0)

**Review by bwpikaard — APPROVED** [review]

**Pull request merged**